usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -2209,12 +2210,19 @@ static CK_RV ep11tok_load_libica(STDLL_TokData_t *tokdata)
         return CKR_OK;
 
     if (strcmp(ep11_data->digest_libica_path, "") == 0) {
-        strcpy(ep11_data->digest_libica_path, ICASHAREDLIB);
+        strcpy(ep11_data->digest_libica_path, ICASHAREDLIB_V4);
         default_libica = 1;
-    }
-
+        libica->library = dlopen(ep11_data->digest_libica_path,
+                                 RTLD_GLOBAL | RTLD_NOW);
```

Comment:
Why here `RTLD_GLOBAL` (in EP11), but not later in ICA?  Is this required/on purpose?

Reply:
No I guess that's not on purpose. Lets remove RTLD_GLOBAL from all dlopen's then....